package.json

```diff
@@ -20,6 +20,7 @@
     "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting"
   },
   "dependencies": {
+    "extend": "^3.0.0",
```

Comment:
that's probably not cool, you got any utility function i can use to __deeply__ merge objects?